packages/build-utils/src/clone-env.ts

```diff
@@ -25,8 +25,13 @@ export function cloneEnv(...envs: (Env | undefined)[]): Env {
       // however we lose this proxied value when we destructure and
       // thus we must explicitly copy it, but we must also remove the
       // `Path` property since we can't have both a `PATH` and `Path`
-      obj.PATH = obj.Path;
-      delete obj.Path;
+
+      // while using git bash obj.path is undefined, but obj.PATH is the correct path
+      if(obj.Path !== undefined){
```

Comment:
Instead of a nested `if`, please combine this condition with the `hasOwnProperty` condition.